protected/humhub/modules/user/models/Password.php

```diff
@@ -198,7 +198,7 @@ private function validateAdvancedPasswordRules($attribute, $params)
             foreach ($additionalRules as $pattern => $message) {
                 $errorMessage = $userModule->isCustomPasswordStrength() ?
                     Yii::t('UserModule.custom', $message) :
```

Comment:
Is this an up to date solution for custom translations? Still not sure whats the suggested approach or translation category for custom user messages.

Reply:
The Translation Manager newly module supports database sources e.g. for profile pields but no support manual config strings.